jr-stree/src/main/java/com/fasterxml/jackson/jr/stree/JacksonJrsTreeCodec.java

```diff
@@ -13,11 +14,25 @@
  */
 public class JacksonJrsTreeCodec extends TreeCodec
 {
-    public static JrsMissing MISSING = JrsMissing.instance;
+    public static final JrsMissing MISSING = JrsMissing.instance;
     protected final ObjectCodec _objectCodec;
 
     // @since 2.17
     protected boolean _failOnDuplicateKeys;
+    protected boolean _useBigDecimalForFloat;
+
+    public JacksonJrsTreeCodec withFeature(JSON.Feature... features) {
```

Comment:
I don't think it's a good idea to require separate configurability of tree codec when `JSON` already has settings. There needs to be a way for codec to access those settings.

So why wouldn't this use the approach already added in #51?

Reply:
My Initial concerns were that we need to use extensions, instead of already present classes.

Reply:
Tree codec is already added as an extension isn't it.